department-of-veterans-affairs/va.gov-cms
# VAGOV-4546: add health service api id field to health service taxonomy

https://va-gov.atlassian.net/browse/VAGOV-4546
Adds the "Health Service API ID" field to the Health Service Taxonomy model.

Note: does not need a feature flag unless consuming on the front-end - this extends the taxonomy content model, not modifies.

QA: 
  - Go to and edit taxonomy content model such as "Audiology": http://va-gov-cms.lndo.site/taxonomy/term/33/edit?destination=/admin/structure/taxonomy/manage/health_care_service_taxonomy/overview
- The "Health Service API ID" should by a text field, right underneath the "Name" field, with help text underneath it
